configcatclient/user.py

```diff
@@ -5,15 +5,60 @@
 from collections import OrderedDict
 from datetime import datetime
 
-from configcatclient import utils
-
 
 class User(object):
     """
-    The user object for variation evaluation
+    User Object. Contains user attributes which are used for evaluating targeting rules and percentage options.
     """
 
     def __init__(self, identifier, email=None, country=None, custom=None):
+        """
+        Initialize a User object.
+
+        Args:
+            identifier: The unique identifier of the user or session (e.g. email address, primary key, session ID, etc.)
+            email: Email address of the user.
+            country: Country of the user.
+            custom: Custom attributes of the user for advanced targeting rule definitions (e.g. role, subscription type, etc.)
+
+            The set of allowed attribute values depends on the comparison type of the condition which references the User
```

Comment:
UX team made changes to this text: https://github.com/configcat/.net-sdk/pull/84/commits/262f33bc4e90ccb376e9595feb897805682ebb9c#diff-102bee679e2207d42ab9004e976a5d3ff4981643b74552379414578d2fa0e4b9